### PR TITLE
Adds Brass Knuckles v2.0

### PR DESCRIPTION
**Pull request by Hubblenaut**
See #563 
Incorporates knuckles into material system.
Available materials are brass (created from steel sheets) and gold.

### CODE REVIEW

**Comment by zaredman** [comment]
What is the difference between this and #563 ?

**Comment by Sabess** [comment]
They're a material item in this. Pull whichever you prefer, I guess; I'll refrain from posting my list of problems with this.

**Comment by Hubblenaut** [comment]
This is free of things left to be fixed and fully incorporated into the material system like it should be.
Sabess, I'm not sure where your problem lies. This is exactly what you have intended to add with the addition that it is embedded in our mechanics.

**Comment by Sabess** [comment]
It's not what I intended. It's what /you/ intended. I'm not going to go into more detail because I'd rather not in a public setting, because I have no gentle way of saying it.

**Comment by Hubblenaut** [comment]
So much for trying to be helpful, I guess

File: code/modules/materials/material_recipes.dm [hunk 1]
```diff
@@ -25,6 +25,8 @@
 
 /material/steel/generate_recipes()
 	..()
+	recipes += new/datum/stack_recipe("brass knuckles", /obj/item/weapon/material/knuckles, 2, time = 10, one_per_turf = 0, on_floor = 1, supplied_material = "brass")
```
Comment (Zuhayr) [inline code/modules/materials/material_recipes.dm]: Making brass knuckles out of steel makes no sense.

**Comment by Zuhayr** [comment]
It's really rude to just outright steamroll over someone's work like that when they have objections to the principle, @Hubblenaut, and it's disingenous to hide behind 'we' when it was specifically you making the decision to do it.

**Comment by Sabess** [comment]
Pretty much that, Zuh, though you said it a lot nicer than I would have. 
Anyway, objectively, the code here may actually be a bit nicer (assuming the force divisor makes it to the originally intended damage). 
A lot of people have been saying they'd rather this be an autolathe thing. So I'll probably just make it an autolathe thing instead.

**Comment by EmperorJon** [comment]
There's an argument that we can already carry around lots of metal and magically make chairs, so why not brass knuckles. Personally I find that rather silly and whilst I can accept it probably won't change I don't particularly want to see more magical item creation like that, especially not items which are intended purely to be weapons. That's why I'd prefer them being autolathe-produced, or even hacked autolathe only, although for balance purposes that's probably asking for too much for something that isn't in practice particularly useful.

**Comment by Hubblenaut** [comment]
I apologize, Sabess, my intention was not to steamroll your work, but to help you out. I adhered to the item properties you've given in your branch, the sole difference here is that the item is a child of weapon/material. That is important for having the item inherit material properties. It is however only available in brass and, as I found to be a nice addition, gold. If you're not comfortable with this however, I will close this PR and leave you to it.


> Making specifically 'brass' knuckles out of steel makes no sense

That is an issue, yes. But an object made of pure brass coming out of the autolathe wouldn't make much more sense either. I'm indifferent to where this is created - forming knifes and spoons out of sheets with your bare hands is the same problem that could be adressed by a complete overhaul of production processes in the future.

**Comment by Sabess** [comment]
Having an item restricted to "hacked" is basically just a formality; it doesn't really change anything. That said, if we put tac knives under hacked, then yeah, ti should go under hacked.

**Comment by zaredman** [comment]
>my intention was not to steamroll your work, but to help you out. 

Now, call me new to Github, but isn't there a way to highlight/comment/edit code without throwing up a new PR out-right? I can see how this is causing agitation.

Also yea, seconding the 'hacked autolathe' one. Keep all weapons to that please.

**Comment by Hubblenaut** [comment]
Since we're at it, we should probably move all the weapons - forks, knife, spoons, blade ect - to the autolathe. If I'm not mistaken though, it can't choose from different materials to build yet.

**Comment by Techhead0** [comment]
If the only reason for making them out of brass is because 'steel brass knuckles' sounds weird, why not just call them 'steel knuckledusters' instead?

**Pull request closed without merging**